client/src/components/login-signup/login.component.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { email: "", password: "" };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit(event) {
+    console.log(this.state);
```

Comment:
Fine for now as a placeholder submit action. Will need replacing with a controller call at some point.
No change necessary

Reply:
Still only console logs the state. To be updated later